### PR TITLE
Update the `Developers` listing

### LINKED ISSUE #4302: Developers page should be split into active/past
Most of the [developers listed](https://github.com/ppy/osu-wiki/blob/master/wiki/People/The_Team/Developers/en.md) are not active, while we have other (open source contributors) which have helped out far more.

Might be good to move all inactive developers to a section below the current list, and list as "past contributors" or similar.

I'm happy to update this if a structure can be decided upon, or I can provide the information on who is currently active if someone wants to take on this task.

### PR DESCRIPTION
**Pull request by TPGPL**
Closes #4302

### CODE REVIEW

**Review by peppy — APPROVED** [review]
seems great

File: wiki/People/The_Team/Developers/en.md [hunk 1]
```diff
@@ -51,11 +48,62 @@ The osu! development effort is lead by ![][flag_AU] **[peppy](https://osu.ppy.sh
...
+
+*For detailed insights on contributions to the osu! project, refer to [this page](https://github.com/ppy/osu/graphs/contributors "GitHub").*
+
+- ![][flag_BY] [EVAST](https://osu.ppy.sh/users/8195163)
+- ![][flag_CA] [DrabWeb](https://osu.ppy.sh/users/6946022)
+- ![][flag_SA] [frenzibyte](https://osu.ppy.sh/users/14210502)
+- ![][flag_CN] [huoyaoyuan](https://osu.ppy.sh/users/2428732)
+- ![][flag_PL] [spaceman\_atlas](https://osu.ppy.sh/users/3035836)
+
+#### osu! framework
```
Comment (Joehuu) [inline wiki/People/The_Team/Developers/en.md:65]: https://github.com/ppy/osu-framework#osuframework
```suggestion
#### osu!framework
```

File: wiki/People/The_Team/Developers/en.md [hunk 2]
```diff
@@ -51,11 +48,62 @@ The osu! development effort is lead by ![][flag_AU] **[peppy](https://osu.ppy.sh
...
+
+*For detailed insights on contributions to the osu! framework project, refer to [this page](https://github.com/ppy/osu-framework/graphs/contributors "GitHub").*
```
Comment (Joehuu) [inline wiki/People/The_Team/Developers/en.md:67]: https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#for-see
```suggestion
*For detailed insights on contributions to the osu!framework project, see: [this page](https://github.com/ppy/osu-framework/graphs/contributors "GitHub")*
```
Not sure about "this page", I thought that link text should be avoided.

File: wiki/People/The_Team/Developers/en.md [hunk 3]
```diff
@@ -51,11 +48,62 @@ The osu! development effort is lead by ![][flag_AU] **[peppy](https://osu.ppy.sh
...
+
+- ![][flag_SA] [frenzibyte](https://osu.ppy.sh/users/14210502)
+- ![][flag_CN] [huoyaoyuan](https://osu.ppy.sh/users/2428732)
+- ![][flag_AU] [Samah](https://osu.ppy.sh/users/343490)
+- ![][flag_PL] [spaceman\_atlas](https://osu.ppy.sh/users/3035836)
+- ![][flag_RU] [TacoToucan](https://osu.ppy.sh/users/1326350)
+
+#### osu! web
```
Comment (Joehuu) [inline wiki/People/The_Team/Developers/en.md:75]: https://github.com/ppy/osu-web#osuweb
```suggestion
#### osu!web
```

File: wiki/People/The_Team/Developers/en.md [hunk 4]
```diff
@@ -51,11 +48,62 @@ The osu! development effort is lead by ![][flag_AU] **[peppy](https://osu.ppy.sh
...
+
+*For detailed insights on contributions to the osu! web project, refer to [this page](https://github.com/ppy/osu-web/graphs/contributors "GitHub").*
```
Comment (Joehuu) [inline wiki/People/The_Team/Developers/en.md:77]:
```suggestion
*For detailed insights on contributions to the osu!web project, see: [this page](https://github.com/ppy/osu-web/graphs/contributors "GitHub")*
```

File: wiki/People/The_Team/Developers/en.md [hunk 5]
```diff
@@ -51,11 +48,62 @@ The osu! development effort is lead by ![][flag_AU] **[peppy](https://osu.ppy.sh
 
 - ![][flag___] [osu!team](https://osu.ppy.sh/users/4341397)
 
+### Contributors
+
+Currently, the main osu! projects are hosted on GitHub and available for anyone to contribute. These are the users who have contributed significantly to these projects and played a vast role in their development.
+
+#### osu!
+
+*For detailed insights on contributions to the osu! project, refer to [this page](https://github.com/ppy/osu/graphs/contributors "GitHub").*
```
Comment (Joehuu) [inline wiki/People/The_Team/Developers/en.md:57]:
```suggestion
*For detailed insights on contributions to the osu! project, see: [this page](https://github.com/ppy/osu/graphs/contributors "GitHub")*
```

**Review by Joehuu — COMMENTED** [review]
For see hatnotes still have a period at the end.

**Review by Joehuu — COMMENTED** [review]
Looks good after this.

**Review by Joehuu — APPROVED** [review]

**Pull request merged**